### ISSUE TITLE
Code licenses

### ISSUE THREAD

**Issue by peastman** [deepchem/torchchem#12]
It looks like a lot of the code here is copied directly from other repositories, but it doesn't include any license notices.

**Comment by rbharath** [comment]
Yes, my bad, was meaning to copy those in later. I believe the current sources of code are as follows:

- DeepChem contrib (the fully connected networks and graph convs are from there).
- https://github.com/gmum/MAT: This is the molecular attention transformer code. It looks like there isn't a license in that repo. I'll raise an issue there and link here to track.

**Comment by rbharath** [comment]
I've raised an issue at https://github.com/gmum/MAT/issues/5.

**Comment by rbharath** [comment]
MAT just added a MIT license so both our sources are MIT licensed. I'll add an MIT license to this repo in a future PR
